sql/testdata/datetime

```diff
@@ -42,6 +42,99 @@ statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\)
 INSERT INTO t VALUES
   ('2015-08-29 20:34:45.34567-07:00'::timestamp, '2015-08-31'::date, '35h2s'::interval)
 
+statement ok
+SET TIME ZONE 'Europe/Rome'
+
+query TTT
+SELECT * FROM t WHERE a = '2015-08-25 05:45:45.53453 CET'::timestamp
+----
+2015-08-25 04:45:45.53453 +0000 UTC   2015-08-25 00:00:00 +0000 UTC   2h45m2.234s
+
+statement ok
+SET TIME ZONE "America/New_York"
+
+query TTT
+SELECT * FROM t WHERE a = '2015-08-24 23:45:45.53453 EST'::timestamp
+----
+2015-08-25 04:45:45.53453 +0000 UTC   2015-08-25 00:00:00 +0000 UTC   2h45m2.234s
+
+# A missing named time zone doesn't result in the use of the set time zone. 
```

Comment:
Yeah, this is weird. The whole point of setting a time zone is to control the time zone that's used for timestamps without  a specified zone.